test/sessionEventAndCoC.js

```diff
@@ -0,0 +1,273 @@
+
+'use strict';
+
+const assert = require('chai').assert;
+let generator = require('../src/backend/generator.js');
+let app = require('../src/app');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['app' is never reassigned. Use 'const' instead. (prefer-const)](https://app.codacy.com/app/triplez/open-event-webapp/pullRequest?prid=3226522)